README.md

```diff
@@ -12,78 +16,92 @@ Some paragraphs are considered "internal", and only meant to be used inside othe
-
-```bash
-composer require openeuropa/oe_whitelabel
-```
+Website projects can use `oe_whitelabel` either as an active theme, or they can create a custom theme using `oe_whitelabel` as a base theme.
 
-### Enable the theme
+### Requirements
 
-In order to enable the theme in your project perform the following steps:
+The package is meant for Drupal projects that manage their dependencies with [Composer](https://www.drupal.org/docs/develop/using-composer/using-composer-to-manage-drupal-site-dependencies#managing-contributed).
 
-- Enable the OpenEuropa Whitelabel Theme and set it as default ```./vendor/bin/drush config-set system.theme default oe_whitelabel_theme```
+Ideally this project should be managed with [Docker](https://www.docker.com/get-docker) and [Docker Compose](https://docs.docker.com/compose/), but this is not a hard requirement.
 
-### Integration with oe_paragraphs
+Check the [composer.json](composer.json) for required PHP version and other dependencies.
 
-In order to have full working integration with paragraphs in your project, you must enable oe_whitelabel_paragraphs module:
+### Add the composer package
 
-```./vendor/bin/drush en oe_whitelabel_paragraphs```
+Add this manually in composer.json, or combine with existing entries:
 
-## Development setup
-
-OpenEuropa Whitelabel theme uses [Webpack](https://webpack.js.org) to compile and bundle SASS and JS.
-
-Make sure you have Node and npm installed.
+```
+    "extra": {
+        "artifacts": {
+            "openeuropa/oe_bootstrap_theme": {
+                "dist": {
+                    "url": "https://github.com/{name}/releases/download/{pretty-version}/{project-name}-{pretty-version}.zip",
+                    "type": "zip"
+                }
+            },
+            "openeuropa/oe_whitelabel": {
+                "dist": {
+                    "url": "https://github.com/{name}/releases/download/{pretty-version}/{project-name}-{pretty-version}.zip",
+                    "type": "zip"
+                }
+            }
+        }
+    }
+```
 
-You can read a guide on how to install node here: https://docs.npmjs.com/getting-started/installing-node
+While this package is still in its `alpha` phase, you need an extra step in composer, to avoid getting anything from the obsolete and unsupported `0.x` branch.
 
-If you prefer to use [Yarn](https://yarnpkg.com) instead of npm, install Yarn by following the guide [here](https://yarnpkg.com/docs/install).
+One option is to set `minimum-stability` and require the `^1.0` version.
+You should also set `prefer-stable` to mitigate the impact on other dependencies.
+Review this again when the site goes into production.
 
-To install required Node.js dependencies run:
+```bash
+composer config minimum-stability alpha
+composer config prefer-stable true
+composer require openeuropa/oe_whitelabel:^1.0
```

Comment:
While technically it is enough to specify `^1.0`, we might still want to write `^1.0@alpha` to be more explicit. TBD.

Reply:
I think it would be overkill to specify ``^1.0@alpha`` since it is ready mentioning that the ``minimum-stability`` is alpha.
I am ok with this current version.